packages/testing/src/server.ts

```diff
@@ -4,7 +4,7 @@ import { promisify } from 'util'
 
 import getStream from 'get-stream'
 
-type Handler = { path: string; response: unknown; status?: number }
+type Handler = { path: string; response?: object; status?: number; wait?: number }
```

Comment:
`wait` is already used in the test suite and in this file, it just wasn't listed in these types